diagnostic.md

```diff
@@ -6,21 +6,22 @@ Record your responses inside the fenced code blocks below each question.
     and what are the main task(s) you perform inside an Ember Route?
 
     ```md
-    <!-- your response here -->
+      The Application Router specifies the URL for every viewstate of your app.
+      An Ember Route loads the model associated with a given route
     ```
 
 1.  What is the command to generate a route named `boston` nested under
     `campus`?
 
     ```md
-    <!-- your response here -->
+      ember generate route campus/boston
     ```
 
 1.  Suppose you have a nested route at the URL `/campus/boston`. How would you
     use the `link-to` helper to generate an appropriate link?
 
     ```md
-    <!-- your response here -->
+      {{#link-to 'boston'}}Boston{{/link-to}}
```

Comment:
We do need to include campus.boston instead of just boston:
`{{#link-to 'campus.boston'}}Boston{{/link-to}}`